opentelemetry-collector.spec.in

```diff
@@ -103,7 +103,9 @@ fi
 %{_bindir}/*
 
 %changelog
-* Thu Aug 28 2024 Benedikt Bongartz <bongartz@redhat.com> - 0.102.1
+* Thu Aug 28 2024 Pavol Loffay <ploffay@redhat.com> - 0.107.0
```

Comment:
not sure if this should be in the future

Reply:
Nope, my bad I copied it and didnt corrent. 

done